.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-02T12:39:22Z" title="Tuesday, April 2nd 2019, 2:39:22 pm +02:00">Apr 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
```


@ulfgebhardt I would not like to encourage people to submit questions via issues. For that we have Discord: https://discordapp.com/invite/6ub73U3

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-02T12:57:12Z" title="Tuesday, April 2nd 2019, 2:57:12 pm +02:00">Apr 2, 2019</time>_

---


I disagree - discord is chaotic and for chatting - while a question raised here has a more permanent nature. Therefore I suggest to implement it. A valid strategy to deal with questions would be
a: link to a part of the documentation
b: create documentation and then link

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-02T12:58:36Z" title="Tuesday, April 2nd 2019, 2:58:36 pm +02:00">Apr 2, 2019</time>_

---


On a side note: The Discord invite is stated in the template to let people think about hooking up there to get their question answered.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-02T15:24:24Z" title="Tuesday, April 2nd 2019, 5:24:24 pm +02:00">Apr 2, 2019</time>_

---


If you disagree, I'm fine with it. Let's merge this and observe how many questions we get through the template. If we are unhappy with the outcome (stale issues, unclear definition-of-done) we can remove that issue template any time.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-04-02T18:39:45Z" title="Tuesday, April 2nd 2019, 8:39:45 pm +02:00">Apr 2, 2019</time>_

---


I like the points that @ulfgebhardt brought up, I am a little nervous it could get out of hand as @roschaefer maybe is as well, but I think it's a nice compromise to deal with the issue if and when it comes up, not before